addon/classes/Column.js

```diff
@@ -129,11 +130,37 @@ export default class Column extends EmberObject.extend({
 
   /**
    * The minimum width (in px) that this column can be resized to.
-   * @property minResizeWidth
+   * @property _minResizeWidth
    * @type {Number}
    * @default 0
+   * @private
+   */
+  _minResizeWidth: 0,
+
+  /**
+   * The minimum width that this column can be resized to.
+   * @property minResizeWidth
+   * @type {String}
+   * @default '0px'
    */
-  minResizeWidth: 0,
+  minResizeWidth: computed({
+    get() {
+      return this.get('_minResizeWidth');
+    },
+    set(key, value) {
+      if (typeOf(value) === 'string') {
```

Comment:
```js
typeof value === 'string'
```

is a bit faster. The only advantage of [`typeOf(item)`](https://github.com/emberjs/ember.js/blob/v2.18.0/packages/ember-runtime/lib/utils.js#L62) is that it correctly detects `String` object instances as `string`, i.e. `typeOf(new String('foo')) === 'string'`.

Personally idc a lot about this scenario, but that just might be because this has never bitten me yet. Keep this line whatever you feel the most comfortable with. :wink: